dspace-server-webapp/src/main/java/org/dspace/app/rest/ItemUploadController.java

```diff
@@ -109,7 +110,7 @@ public BitstreamResource uploadBitstream(HttpServletRequest request, @PathVariab
             log.error(message, e);
             throw new RuntimeException(message, e);
         }
-        return new BitstreamResource(bitstreamConverter.fromModel(bitstream), utils);
+        return converter.toResource(converter.toRest(bitstream, Projection.DEFAULT));
```

Comment:
Again, curious why is this a hardcoded Projection instead of `utils.obtainProjection()`?